---
id: 363335a1ea
question: Error index 311297 is out of bounds for axis 0 with size 131483 when loading parquet file.
section: Module 4: Deployment
course: mlops-zoomcamp
sort_order: 1620
---

Problem description:   Running starter.ipynb in homeworkâ€™s Q1 will show up this error.

Solution description: Update pandas (actually pandas version was the latest, but several dependencies are updated).

Added by Marcos Jimenez

